tests/unit/APITest.ts

```diff
@@ -1,5 +1,6 @@
-import Onyx from 'react-native-onyx';
-import _ from 'underscore';
+// import Onyx from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import reactNativeOnyxMock from '../../__mocks__/react-native-onyx';
```

Comment:
```suggestion
import type {ReactNativeOnyxMock} from '../../__mocks__/react-native-onyx';
```

Export the `ReactNativeOnyxMock` in that file.